HelloWorld/project/projectx.cs

```diff
@@ -0,0 +1,47 @@
+using System;
```

Comment:
This whole file causes your project not to run. Please delete this file, or comment out every line in it.

With .NET Core, the `.csproj` file assumes every other file that is in the same directory as or one of it's children belongs in project and should be compiled. I will go into more detail about this on Wednesday.

Reply:
I have updated the file and resubmitted pull request.